src/common/tools/time.cpp

```diff
@@ -92,4 +96,35 @@ StaticString<cTimeStrLen> Duration::ToISO8601String() const
     return result;
 }
 
+/***********************************************************************************************************************
+ * Time
+ **********************************************************************************************************************/
+
+/***********************************************************************************************************************
+ * Public
+ **********************************************************************************************************************/
+
+RetWithError<StaticString<cTimeStrLen>> Time::ToUTCString() const
+{
+    tm                        buf;
+    StaticString<cTimeStrLen> utcTimeStr;
+
+    auto unixTime = UnixTime();
+
+    auto time = gmtime_r(&unixTime.tv_sec, &buf);
+
+    utcTimeStr.Resize(utcTimeStr.MaxSize());
+
+    size_t size = strftime(utcTimeStr.Get(), utcTimeStr.Size(), "%FT%TZ", time);
+    if (size == 0) {
+        return {{}, ErrorEnum::eFailed};
```

Comment:
maybe here to add message like can't convert to UTC Time and on called side print err ?